scipy/stats/tests/test_qmc.py

```diff
@@ -312,23 +312,23 @@ def test_NormalQMCSeededInvTransform(self):
         engine = qmc.NormalQMC(dim=2, seed=12345, inv_transform=True)
         samples = engine.random(n_samples=2)
         samples_expected = np.array(
-            [[-0.41622922, 0.46622792], [-0.96063897, -0.75568963]]
+            [[ 0.228309, -0.162516], [-0.41622922, 0.46622792]]
         )
         assert_array_almost_equal(samples, samples_expected)
         # test odd dimension
         engine = qmc.NormalQMC(dim=3, seed=12345, inv_transform=True)
         samples = engine.random(n_samples=2)
         samples_expected = np.array(
             [
+                [0.228309, -0.162516, 0.167352],
                 [-1.40525266, 1.37652443, -0.8519666],
-                [-0.166497, -2.3153681, 0.840378],
             ]
         )
         assert_array_almost_equal(samples, samples_expected)
 
     def test_NormalQMCShapiro(self):
         engine = qmc.NormalQMC(dim=2, seed=12345)
-        samples = engine.random(n_samples=250)
+        samples = engine.random(n_samples=350)
```

Comment:
may make sense to use base-2 numbers (here and throughout the rest of the code)? e.g. 512